buildspecs/prod-release-nodeadm.yml

```diff
@@ -3,14 +3,12 @@ version: 0.2
 phases:
   pre_build:
     commands:
+      # ensure staging release artifacts are valid before starting prod release
+      - ./hack/validate-release-artifacts.sh $STAGING_BUCKET latest
       - echo "Downloading artifacts from staging bucket..."
-      - aws s3 sync --no-progress s3://${STAGING_BUCKET}/latest/linux/amd64/ _bin/amd64/
-      - aws s3 sync --no-progress s3://${STAGING_BUCKET}/latest/linux/arm64/ _bin/arm64/
-      - aws s3 cp --no-progress s3://${STAGING_BUCKET}/latest/GIT_VERSION _bin/GIT_VERSION
-
-      - export VERSION=$(cat _bin/GIT_VERSION)
-      - echo "Using version:"
-      - cat _bin/GIT_VERSION
+      - aws s3 sync --no-progress s3://${STAGING_BUCKET}/latest/ ./latest
+      - export VERSION=$(cat latest/GIT_VERSION)
+      - echo "Using version: ${VERSION}"
```

Comment:
Buildspec env variable expansion wont work, it'll fail deployment. We can cat the file though.

Reply:
fixed